### PR TITLE
builds page: capitalize tabs

### PR DESCRIPTION
**Pull request by grahamc**
Minor UI fixup.

### CODE REVIEW

File: src/root/build.tt [hunk 1]
```diff
@@ -145,10 +145,11 @@ END;
...
-  [% IF steps.size() > 0 %]<li class="nav-item"><a class="nav-link" href="#tabs-buildsteps" data-toggle="tab">Build steps</a></li>[% END %]
-  [% IF build.dependents %]<li class="nav-item"><a class="nav-link" href="#tabs-usedby" data-toggle="tab">Used by</a></li>[% END%]
-  [% IF drvAvailable %]<li class="nav-item"><a class="nav-link" href="#tabs-build-deps" data-toggle="tab">Build dependencies</a></li>[% END %]
-  [% IF localStore && available %]<li class="nav-item"><a class="nav-link" href="#tabs-runtime-deps" data-toggle="tab">Runtime dependencies</a></li>[% END %]
+  [% IF steps.size() > 0 %]<li class="nav-item"><a class="nav-link" href="#tabs-buildsteps" data-toggle="tab">Build Steps</a></li>[% END %]
+  [% IF build.dependents %]<li class="nav-item"><a class="nav-link" href="#tabs-usedby" data-toggle="tab">Used By</a></li>[% END%]
+  [% IF drvAvailable %]<li class="nav-item"><a class="nav-link" href="#tabs-build-deps" data-toggle="tab">Build Dependencies</a></li>[% END %]
+  [% IF available %]<li class="nav-item"><a class="nav-link" href="#tabs-runtime-deps" data-toggle="tab">Runtime Dependencies</a></li>[% END %]
```
Comment (grahamc) [inline src/root/build.tt:151]: Not sure how this line snuck in here.

**Comment by samueldr** [comment]
either *[Title Case](https://en.wikipedia.org/wiki/Title_case)* or *[Sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)* can be used, I guess, but it should stay consistent overall in the app.

Is there a particular reason for migrating to *Title Case* here? Nothing in the app AFAICT quickly uses *Title Casing*.

**Comment by samueldr** [comment]
(It could be *[Start Case](https://en.wikipedia.org/wiki/Letter_case#Start_case)* I guess)

**Comment by grahamc** [comment]
It just looked nicer to me :sweat_smile:. I could, uh, not do this. I could also go make it more consistent elsewhere. What do you think?

**Comment by samueldr** [comment]
*Title Case* is not the law, and really is more for published works, or works of art. E.g. thesis paper title, or book name.

* * *

Though I don't know if there are any serious UX guidelines around title casing and *sections* of apps.

It seems [Microsoft once suggested using *Title Case* for tab titles](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/bb246428(v=vs.85)?redirectedfrom=MSDN).

The [GNOME HIG suggests it too](https://developer.gnome.org/hig/guidelines/writing-style.html).

Similarly [the macOS HIG suggests using title-style capitalization](https://developer.apple.com/design/human-interface-guidelines/macos/windows-and-views/tab-views/).

* * *

Huh... Looks like it's generally done *for apps*. But we probably need to stay consistent for all tabs, *and* consider proper capitalization fixes if needed elsewhere.

**Comment by samueldr** [comment]
Using this naïve `ag class.*nav-item.*nav-link`, it looks like there's still tabs to capitalize correctly in:

 - src/root/jobset-eval.tt


Everything else is composed solely of single-words already-capitalized tabs it looks like.

**Comment by grahamc** [comment]
Fixup'd, thanks!

**Pull request merged**